main.py

```diff
@@ -87,7 +86,6 @@ def record_word_def_chosen(word, definition, word_id):
 # Instantiate EmailMessage class with all necessary info for the email being sent
 em = EmailMessage()
 em['From'] = email_sender
-em['To'] = email_receiver
```

Comment:
The `em['To']` is removed as it is unnecessary as it is simply what is printed on the email in the `To:` section. It is easier to have it included when an email is sent to only one person, but it would require some additional changes to make this header work for more than one email. For the time being, this will be removed. 

See: https://stackoverflow.com/a/8861795